in-work/cobweb.md

```diff
@@ -287,62 +283,91 @@ def plot45(model, pmin, pmax, p0, num_arrows=5):
     plt.show()
 ```
 
-```{code-cell} ipython3
+Now we can set up a market and plot the 45 degree diagram.
+
+```python
+m = Market()
+```
+
+```python
 plot45(m, 0, 9, 2, num_arrows=3)
 ```
 
-The plot shows the function $f$ defined in {eq}`def_f` and the $45$ degree line.
+The plot shows the function $g$ defined in {eq}`def_g` and the $45$ degree line.
 
 Think of $ p_t $ as a value on the horizontal axis.
 
-To calculate $ p_{t+1} $, we use the graph of $f$ to see its value on the vertical axis.
+Since $p_{t+1} = g(p_t)$, we use the graph of $g$ to see $p_{t+1}$ on the vertical axis.
 
 Clearly,
 
-- If $ f $ lies above the 45 degree line at this point, then we have $ p_{t+1} > p_t $.
-- If $ f $ lies below the 45 degree line at this point, then we have $ p_{t+1} < p_t $.
-- If $ f $ hits the 45 degree line at this point, then we have $ p_{t+1} = p_t $, so $ p_t $ is a steady state.
+- If $ g $ lies above the 45 degree line at $p_t$, then we have $ p_{t+1} > p_t $.
+- If $ g $ lies below the 45 degree line at $p_t$, then we have $ p_{t+1} < p_t $.
+- If $ g $ hits the 45 degree line at $p_t$, then we have $ p_{t+1} = p_t $, so $ p_t $ is a steady state.
 
+Consider the sequence of prices starting at $p_0$, as shown in the figure.
 
-```{code-cell} ipython3
-def ts_plot_price(model, p0, ts_length=10):
-    """
-    Function to simulate and plot the time series of price.
+We find $p_1$ on the vertical axis and then shift it to the horizontal axis using the 45 degree line (where values on the two axes are equal).
 
-    Parameters
-    ==========
+Then from $p_1$ we obtain $p_2$ and continue.
 
-    model: Market model
+We can see the start of a cycle.
 
-    p0: Initial value of price
+To confirm this, let's plot a time series.
+
+```python
+def ts_plot_price(model,             # Market model
+                  p0,                # Initial price
+                  y_a=3, y_b= 12,    # Controls y-axis
+                  ts_length=10):     # Length of time series
+    """
+    Function to simulate and plot the time series of price.
 
-    ts_length: Number of iterations
     """
     fig, ax = plt.subplots()
     ax.set_xlabel(r'$t$', fontsize=12)
     ax.set_ylabel(r'$p_t$', fontsize=12)
     p = np.empty(ts_length)
     p[0] = p0
     for t in range(1, ts_length):
-        p[t] = f(model, p[t-1])
+        p[t] = g(model, p[t-1])
     ax.plot(np.arange(ts_length),
             p,
             'bo-',
             alpha=0.6,
             lw=2,
             label=r'$p_t$')
     ax.legend(loc='best', fontsize=10)
+    ax.set_ylim(y_a, y_b)
     ax.set_xticks(np.arange(ts_length))
     plt.show()
 ```
 
-```{code-cell} ipython3
-ts_plot_price(m, 2, 15)
+```python
```

Comment:
Is this an expected change?

Reply:
This isn't a manual change and might be having some issues with `jupytext`. cc @mmcky

Reply:
For context @mmcky , I shifted this backward and forward with jupytext but did not edit that part manually.

(Come to think about it, maybe I accidentally did `jupytext --to md` instead of `--to myst`...)

Reply:
Thanks @Smit-create and @jstac this won't matter for `jupyter-book` as we have defined synonyms for `code-cell`. I will check this once we migrate them over.

Reply:
Actually upon closer inspection:

````
```python
````

will cause an issue with code execution. 

@Smit-create would you mind to update them back to using `code-cell` today?